core/modules/dblog/tests/dblog.test

```diff
@@ -46,6 +46,9 @@ class DBLogTestCase extends BackdropWebTestCase {
 
     $row_limit = 1000;
     $this->verifyRowLimit($row_limit);
+    // Remove seconds from date so this test is more likely to always pass.
+    $date_format = 'M d, Y -  j:ia';
```

Comment:
Throughout this test, there is an extra space in the date format between the dash and j: `-  j`

Reply:
fixed.